### ISSUE TITLE
Why this test case passed?

### ISSUE THREAD

**Issue by jukanntenn** [NaturalSelectionLabs/RSS3-SDK-for-JavaScript#2]
```javascript
test('Account.sign with singer', () => {
    const signer = ethers.Wallet.createRandom();
    const rss3 = new RSS3({
        endpoint: '',
        mnemonic: signer.mnemonic.phrase,
    });
    expect(
        rss3.account.sign({
            agent_id: 'test',
            test1: 'r',
        }),
    ).toEqual(
        signer.signMessage(
            object.stringifyObj({
                test1: 'r',
            }),
        ),
    );
});
```

It seems that `Account.sign` returns nothing while `signer.signMessage` return the signature it signed. Could the results to be equal?

**Comment by DIYgod** [comment]
You are right, it was fixed in 93dca7b5f0bce80167cd7757a0461027c1f56d1b

It passed because they both return a promise, which is pointless.

**Comment by jukanntenn** [comment]
That's cool. So for jest, Promise<void> equals Promise<string>, at least in this test case.
